File: 3830-find-closest-person/find-closest-person.cs
public class Solution 
{
    public int FindClosest(int x, int y, int z) 
    {
        int d1 = Math.Abs(z - x);
        int d2 = Math.Abs(z - y);

        if (d1 < d2) return 1;
        else if (d1 > d2) return 2;
        return 0;
    }
}

File: 3830-find-closest-person/find-closest-person.java
class Solution 
{
    public int findClosest(int x, int y, int z) 
    {
       int m=Math.abs(z-x);
       int n=Math.abs(z-y);    
       if(m==n)
       {
        return 0;
       }
       if(m>n)
       {
        return 2;
       }
       return 1;
    }
}